src/main/java/Duke.java

```diff
@@ -1,10 +1,115 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        DukeList list = new DukeList();
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String bound = "____________________________________________________________";
+        String topBound = "____________________________________________________________\n";
+        String bottomBound = "\n____________________________________________________________\n";
+
+        System.out.println(topBound + logo + "\nHello! I'm Duke\n" + "What can I do for you?\n" + bottomBound);
+
+        while (sc.hasNextLine()) {
+            String input = sc.nextLine();
+            String[] arr = input.split("/");
+            if (input.equals("bye")) {
+                System.out.println(topBound + "Good bye! Stay calm and keep coding o7" + bottomBound);
+                sc.close();
+                break;
+            } else if (input.equals("list")) {
+                System.out.println(bound);
+                System.out.println("Here are the tasks in your list");
+                list.printAll();
+                System.out.println(topBound);
+            } else if (arr[0].startsWith("done")) {
+                try {
+                    String[] doneArr = arr[0].split(" ");
+                    int x = Integer.parseInt(doneArr[1]) - 1;
+                    list.done(x);
+                    System.out.println(topBound + "Good job! I've marked this task as done:\n" + "  " + list.get(x)
+                            + bottomBound);
+                } catch (IndexOutOfBoundsException e) {
+                    // Task number x does not exist
+                    System.out.println(topBound + "Task does not exist and can not be completed!" + bottomBound);
+                } catch (NumberFormatException e) {
+                    // parameter for done is not an Integer
+                    System.out.println(topBound + "done must be followed by an Integer ie. done 1. Try again!"
+                        + bottomBound);
+                }
```

Comment:
I have noticed that you have implemented DukeException.java as a error handling class. Perhaps you can leverage that to handle your exceptions instead of printing them here?